nflows/transforms/splines/rational_quadratic.py

```diff
@@ -34,16 +33,18 @@ def unconstrained_rational_quadratic_spline(
         constant = np.log(np.exp(1 - min_derivative) - 1)
         unnormalized_derivatives[..., 0] = constant
         unnormalized_derivatives[..., -1] = constant
-
-        outputs[outside_interval_mask] = inputs[outside_interval_mask]
-        logabsdet[outside_interval_mask] = 0
+        
+        outputs = torch.where(outside_interval_mask, inputs, outputs)
```

Comment:
Should we drop the zero init [above](https://github.com/bayesiains/nflows/pull/77/commits/04989c4a65c70c6cda29cf8c645140f98f383b59#diff-56a605777531dad36582c6a82ec8a384705f36c6c342215093586bdfadd58198R28-R29) if we're copying here anyway? I.e. `outputs = torch.where(outside_interval_mask, inputs, torch.zeros_like(inputs))`.